### ISSUE TITLE
doc header and footer data support

### ISSUE THREAD

**Issue by sxj2024** [WhityGhost/gh0ffice#1]
Hey buddy, can you support parsing the header and footer content of the doc document

**Comment by WhityGhost** [comment]
yes, but it could take too much time for you to see the next version
this project isn't maintained at the moment
